### PR TITLE
Fix to tracer-http after testing with collector.

### PR DESCRIPTION
**Pull request by MatthewDolan**

### CODE REVIEW

File: example/src/test/java/com/lightstep/tracer/shared/ExampleTracerTest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import io.opentracing.Span;
+import org.junit.Test;
+
+public class ExampleTracerTest {
+    @Test
+    public void test() throws Exception {
+        ExampleTracer tracer = new ExampleTracer(new Options.OptionsBuilder()
+                .withAccessToken("LIGHTSTEP_LOCAL_DEV_ACCESS_TOKEN")
+                .withCollectorHost("127.0.0.1")
+                .withCollectorPort(9996)
```
Comment (danielamiao) [inline example/src/test/java/com/lightstep/tracer/shared/ExampleTracerTest.java]: to be clear this is the part I'm not sure about - change to public collector host/port?

Comment (MatthewDolan) [reply example/src/test/java/com/lightstep/tracer/shared/ExampleTracerTest.java]: Sure. Should I create a fake access token or something?

Comment (MatthewDolan) [reply example/src/test/java/com/lightstep/tracer/shared/ExampleTracerTest.java]: FYI, I am following the convention of https://github.com/lightstep/lightstep-tracer-go/blob/master/examples/trivial.go#L49, maybe I should change it from a test to a function with a main instead.

**Comment by MatthewDolan** [comment]
Thinking about it some more, I would prefer to keep 127.0.0.1 for now. I don't want to create an external project and have put an access token right here into the code.

**Comment by danielamiao** [comment]
Dang we are inconsistent across libraries basically, I was thinking of the JRE example code: https://github.com/lightstep/lightstep-tracer-java/blob/master/examples/src/main/java/com/lightstep/tracer/jre/example/Simple.java

**Comment by MatthewDolan** [comment]
@danielamiao I was convinced :). I fixed it!

**Pull request merged**